requirements.api.txt

```diff
@@ -5,4 +5,5 @@ s3path==0.3.2
 scikit-learn==0.24.2
 boto3==1.17.112
 google-cloud-bigquery==2.20.0
+requests==2.26.0
```

Comment:
it might be worth moving to the dev dependencies if requests is only used by the tests?